src/style.css

```diff
@@ -0,0 +1,66 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav__logo {
+  object-fit: contain;
+  object-position: center;
+}
+
+.nav__list {
+  display: flex;
+  align-self: center;
+  margin: 0;
+}
+
+.nav__item {
+  display: inline;
+}
+
+.nav__link {
+  position: relative;
+  display: inline-block;
+  text-decoration: none;
```

Comment:
what happens if you remove text-decoration: none?

Reply:
nothing. deleted this rule